build.gradle

```diff
@@ -9,6 +17,9 @@ apply plugin: 'maven'
 apply plugin: 'maven-publish'
 apply plugin: 'antlr'
 
+apply plugin: 'osgi'
```

Comment:
Actually, that's exactly what I get with the osgi plugin only, without biz.aQute.bnd.builder (only in my case the version of bnd tool differs when I include biz.aQute.bnd.builder,  it's Bnd-3.1.0.201512181341). So including it or not does not change anything, better keep osgi plugin only.